nf1_cellprofiler_classifier_fs_all_cellprofiler_all_feat_perfect.py

```diff
@@ -168,24 +203,50 @@ def kcross_val(model, feat, splits=num_splits):
 # In[15]:
 
 
-# Returns the naive accuracy:
-def naive_acc(labels):
-    naive_pred = rng.integers(low=0, high=2, size=labels.shape[0])
-    naive_acc = accuracy_score(naive_pred,labels)
-    return naive_acc
+# Naive Model:
+class naive_model:
```

Comment:
Consider adding a class docstring to help describe what this class does.

Reply:
Added!